timesketch/frontend-ng/src/components/Explore/EventTags.vue

```diff
@@ -0,0 +1,91 @@
+<!--
+Copyright 2023 Google Inc. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+  <span>
+    <v-chip
+      small
+      class="mr-1"
+      v-for="tag in sortedTags"
+      :close="showDetails ? true : false"
```

Comment:
could we wrap this in a `<hover>` instead?

Reply:
Yeah, that works as well. However, with the hover option the table jumps a bit around when hovering. Please have a look and let me know what you think.

Reply:
fixed by adding a 32px padding on non-hover.